dev/package.json

```diff
@@ -13,17 +13,20 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router": "^5.2.0",
-    "react-router-dom": "^5.3.0"
+    "react-router-dom": "^5.3.0",
+    "selenium-webdriver": "^4.1.1"
```

Comment:
Must the `selenium-webdriver` and `@wojtekmaj/enzyme-adapter-react-17` be part of the "actual" dependencies instead of just dev-dependencies??